packages/contracts/test/temp.spec.ts

```diff
@@ -2,7 +2,7 @@ import { Contract, providers } from "ethers";
 
 import artifact from "../artifacts/contracts/test/TestERC20.sol/TestERC20.json";
 
-describe("Sample", function () {
+describe.skip("Sample Script", function () {
   const BscUSDC = "0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d";
 
   const rpcUrl = "https://bsc-dataseed.binance.org";
```

Comment:
Can we delete this whole file or is this for testing against remote chains? If so, we shouldn't be executing only some test suites, and instead we should be executing the *entire* test suite to make sure that it can just be pointed at some testnet and we can run it

Reply:
removing this file